mmv1/third_party/terraform/resources/resource_composer_environment.go.erb

```diff
@@ -433,6 +434,23 @@ func resourceComposerEnvironment() *schema.Resource {
 								},
 							},
 						},
+						"encryption_config": {
+							Type:         schema.TypeList,
+							Optional:     true,
+							Computed:     true,
+							AtLeastOneOf: composerConfigKeys,
+							MaxItems:     1,
+							Description:  `The encryption options for the Composer environment and its dependencies.`,
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+									"kms_key_name": {
+										Type:         schema.TypeString,
+										Required:     true,
+										Description: `Optional. Customer-managed Encryption Key available through Google's Key Management Service. Cannot be updated.`,
```

Comment:
If this cannot be updated as indicated by the description, it should probably be marked `ForceNew: true` to force Terraform to recreate the resource if this field is changed.

Reply:
Thanks, added!

With regard to tests - I've noticed some problems with deletion after the test is complete (a firewall rule related issue, which, based on the comments, is known to happen), but nothing at creation.